src/main/java/sggw/wzim/czasnawypad/db/dto/AttractionRatingDTO.java

```diff
@@ -20,11 +20,13 @@
 @EqualsAndHashCode
 public class AttractionRatingDTO {
     private Integer id;
+    private Integer userId;
```

Comment:
Why do we expose `userId` here?

Reply:
Seba prosił bo mu się przyda przy sprawdzeniu czy dla danej atrakcji user już utworzył rating